tests/lib/prepare-restore.sh

```diff
@@ -608,6 +608,15 @@ prepare_project() {
         disable_journald_rate_limiting
         disable_journald_start_limiting
     fi
+
+    # native jq replacement, but still with some incompatibilies, see
+    # https://github.com/itchyny/gojq
+    # major differences:
+    # - map keys are sorted by default
+    # - with --yaml-input, can parse YAML
+    GOBIN=$PROJECT_PATH/tests/bin \
+    CGO_ENABLED=0 \
+        go install github.com/itchyny/gojq/cmd/gojq@v0.12.16
```

Comment:
We might consider just pulling one of the pre-built artefacts instead:

- https://github.com/itchyny/gojq/releases/download/v0.12.16/gojq_v0.12.16_linux_amd64.tar.gz
- https://github.com/itchyny/gojq/releases/download/v0.12.16/gojq_v0.12.16_linux_arm64.tar.gz

The build could be only done on arch mismatch.

Reply:
are those built with CGO disabled?